cmd/remotefs/azurefs.go

```diff
@@ -266,7 +266,7 @@ func releaseRemoteFilesystemKey(tempDir string, keyDerivationBlob common.KeyDeri
 			return "", errors.Wrapf(err, "failed to derive oct key")
 		}
 
-		logrus.Debugf("Symmetric key %s (salt: %s label: %s)", hex.EncodeToString(octetKeyBytes), keyDerivationBlob.Salt, labelString)
+		logrus.Debugf("Symmetric key: (salt: %s label: %s)", keyDerivationBlob.Salt, labelString)
```

Comment:
There is no longer a key after "key:" which might confuse someone reading this log in a "was the key empty?" sort of a way.

Reply:
Changed to:
```
		logrus.Debugf("Symmetric key salt: %s label: %s", keyDerivationBlob.Salt, labelString)
```